plugins.mk

```diff
@@ -25,7 +25,8 @@ LFE_FILES = $(sort $(call core_find,src/,*.lfe))
 
 ifneq ($(LFE_FILES),)
 
-BEAM_FILES += $(addprefix ebin/,$(patsubst %.lfe,%.beam,$(notdir $(LFE_FILES))))
+BEAM_FILES += $(addprefix ebin/, $(addsuffix .beam,\
+	$(shell grep -h '^.defmodule ' src/*.lfe | sed 's/^.defmodule *\([^ ]*\).*/\1/')))
```

Comment:
The regexp should be `^\(defmodule `, but I hadn’t managed to escape the parenthesis, so have replaced it with a dot.